mcp/pacemaker.in

```diff
@@ -90,6 +91,13 @@ if [ x = "x$PCMK_STACK" -a -f "@sysconfdir@/cluster/cluster.conf" ]; then
     PCMK_STACK=cman
 fi
 
+# Check if there is a valid watchdog-device configured in sbd config
+if [ x != "x$SBD_WATCHDOG_DEV" -a "/dev/null" != "$SBD_WATCHDOG_DEV" -a -c "$SBD_WATCHDOG_DEV" ]; then
+	if chkconfig --list sbd_helper 2>/dev/null | grep -q ":on"; then
+		SBD_SERVICE=sbd_helper
+	fi
+fi
+
```

Comment:
Unfortunately you probably can as well have chkconfig without cman ...
Actually the code behaves exactly the same if you guard it with some checking for chkconfig
or if you don't - not even a nag output ...
But of course for showing that we are aware that it might not be there we can add the guard.